e2etests/testdata/regression/unconnected/elk/board.exp.json

```diff
@@ -531,7 +531,6 @@
       "underline": false,
       "labelWidth": 639,
       "labelHeight": 51,
-      "labelPosition": "INSIDE_MIDDLE_CENTER",
```

Comment:
why did this go away?

Reply:
i this it's becasue i removed this code snippet

<img width="943" alt="image" src="https://user-images.githubusercontent.com/19243114/228098026-773d7ef6-02af-4cb7-ade6-1ddb3955212e.png">

Reply:
any issues to remove this code snippet?

Reply:
I think we need it since based on the comment, it is necessary to position labels for objects with constant near positions

Reply:
ohh i see, but for now only constant text type is allowed right? and set near to another shape is only for TALA engine.

Reply:
oh it looks like it doesn't need to be text type though, the following is valid:

```d2
a.shape: person
a.near: top-left
```
so in this case it would need a labelPosition set, and since its a person it would go to outside bottom center

Reply:
![Screen Shot 2023-04-04 at 8 29 51 PM](https://user-images.githubusercontent.com/85081687/229974097-499af292-0356-4058-aec7-9c0151e63058.png)

it also looks like the current d2 version has a bug where icon position isn't set correctly

```d2
a.shape: person
a.near: top-left
a.icon: https://icons.terrastruct.com/essentials/142-target.svg
```